gettsim/pre_processing/policy_for_date.py

```diff
@@ -60,10 +60,11 @@ def get_policies_for_date(policy_date, policy_groups="all"):
     ----------
     policy_date : int, str, datetime.date
         The date for which the policy system is set up.
-    policy_groups : list, str
+    policy_groups, groups : list, str
         The group or a list of groups which parameters are loaded. If an invalid
         name is given, a list of all possible values is printed. Default is to load all
-        parameter groups.
+        parameter groups. The keyword group will be completly replaced in future
+        versions.
```

Comment:
There exist a special rst role for deprecations. Scroll a little bit further down on this page (https://numpydoc.readthedocs.io/en/latest/format.html#sections). It is not necessary, but nice to try it out.

Reply:
As I wrote on Zulip, nobody used this keyword for sure, no need to leave a trace of it.